demos/breakout_ai/main.py

```diff
@@ -161,15 +150,13 @@ def run(self):
 
                 # paddle_counter += 1
 
-                self.output_queue.put("SCORE " + str(score))
-                self.output_queue.put("LIVES " + str(lives))
-
                 if self.ball[1] <= self.level + 2:
                     row = self.ball[1]
                     if row in self.bricks.keys():
                         if self.ball[0] in self.bricks[row]:
                             is_down = not is_down
                             score += SCORE_INC
+                            self.output_queue.put("SCORE " + str(score))
```

Comment:
I think this is beyond this PR, but I think it would be better to put a dictionary in the queue instead of a string. We could standardize the dictionary to contain certain keys. But like I said, this is beyond the scope of this PR and something we can tackle later. This is related to #12.

Reply:
I agree